composer.json

```diff
@@ -17,28 +17,29 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "ext-json": "*",
-        "graylog2/gelf-php": "^1.4.2",
-        "monolog/monolog": "^1.24 || ^2.0",
-        "sentry/sdk": "^3.1",
-        "sentry/sentry": "^3.1",
-        "sentry/sentry-symfony": "^4.0",
-        "symfony/config": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/dependency-injection": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/expression-language": "^3.0 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^3.4.26|^4.2.7|^5.0|^6.0",
-        "symfony/http-kernel": "^3.4|^4.0|^5.0|^6.0",
-        "symfony/monolog-bundle": "^3.4|^5.0"
+        "graylog2/gelf-php": "^1.4.2 || ^2.0",
+        "sentry/sentry-symfony": "^5.3",
+        "symfony/config": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "symfony/dependency-injection": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "symfony/http-kernel": "^4.4.20||^5.0.11||^6.0||^7.0",
+        "monolog/monolog": "^2.0.1",
+        "symfony/monolog-bundle": "^3.4",
+        "symfony/expression-language": ">=2.6",
+        "jean85/pretty-package-versions": "^2.0"
```

Comment:
What's this for? It's not used anywhere

Reply:
Yeah, basically what tests shows is that when used with lowest dependency versions feature does not work, because `\Jean85\Version::getPrettyVersion` fails to resolve package version. The same can happen in prod, so it's a good catch. If you would look at \Sentry\SentryBundle\DependencyInjection\SentryExtension::loadInternal code, you'd see that call to `Jean85\Version::getPrettyVersion` can be bypassed if `env(SENTRY_RELEASE)` container parameter is set.